new/LIT7142Semon.xml

```diff
@@ -0,0 +1,67 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7142Semon" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ስምዖን፡ ዘዓምድ፡</title>
+                <title xml:lang="gez" type="normalized"  corresp="#t1">Sǝmʻon za-ʻāmd</title>
```

Comment:
please note that for ayn and alif we use ʿ (Unicode 02BF) and ʾ (02BE)  (and not ʻ 02BB and ʼ 02BC which are different unicode symbols used for quotation marks)

https://betamasaheft.eu/Guidelines/?id=transliteration-principles

thank you!